build.gradle

```diff
@@ -35,6 +35,7 @@ allprojects {
 
 	repositories {
 		if (version.endsWith('BUILD-SNAPSHOT') || project.hasProperty('platformVersion')) {
+			mavenLocal()
```

Comment:
Oh! Sure!
That is an artefact for testing against fix in IO, which I have reached with the `mvn install -DskipTests` locally.
Doh...
Feel free to remove on merge.
Thank you!